local/nginx/nginx.conf

```diff
@@ -0,0 +1,34 @@
+user  nginx;
+worker_processes  auto;
+
+error_log  /var/log/nginx/error.log warn;
+pid        /var/run/nginx.pid;
+
+
+events {
+    worker_connections  1024;
+}
+
+
+http {
+    include       /etc/nginx/mime.types;
+    default_type  application/octet-stream;
+
+    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+                      '$status $body_bytes_sent "$http_referer" '
+                      '"$http_user_agent" "$http_x_forwarded_for"';
+
+    access_log  /var/log/nginx/access.log  main;
+
+    sendfile        on;
+    #tcp_nopush     on;
+
+    keepalive_timeout  65;
+
+    gzip on;
+    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript application/octet-stream;
```

Comment:
Why do you prefer to use `expired no-cache no-store private` instead of `any`?

Reply:
Yeah, you are right. I thought this could make sense for Cloudflare, but we shouldn't have anything non cacheable that is important not compress.

Reply:
So, I thought about something else: We should set this not to `any` but to `off`.

If I understand the setting correctly, that would mean that nginx won't compress if the request comes from Cloudflare or another proxy, leaving the compression to them. This could make sense to reduce CPU usage for our nodes, in exchange for bandwidth between Cloudflare and the node, which I think is pretty high.